astropy/coordinates/baseframe.py

```diff
@@ -37,7 +37,7 @@
 # encouraged, as this is not the public API location.)
 from .frame_attributes import (
     TimeFrameAttribute, QuantityFrameAttribute,
-    EarthLocationAttribute, CoordinateAttribute,
+    EarthLocationFrameAttribute, CoordinateFrameAttribute,
```

Comment:
After addressing the comment below (about adding a deprecation warning), revert this change so that only the old class names get imported to baseframe. This is around only for backwards-compatibility, but we want to encourage users to import from `frame_attributes.py` in the future. So the new classes should just exist in there.